src/Textual/blocks/text.jl

```diff
@@ -27,3 +27,11 @@ struct Paragraph <: Block end
 
 FastAI.checkblock(::Paragraph, ::String) = true
 FastAI.mockblock(::Paragraph) = randstring(" ABCEEFGHIJKLMNOPQESRUVWXYZ 1234567890 abcdefghijklmnopqrstynwxyz\n\t.,", rand(10:40))
+
+struct TokenVector <: Block end
+
+FastAI.checkblock(::TokenVector, ::Vector{String}) = true
```

Comment:
Let's call this block `Tokens`, to make clear it's multiple tokens. These blocks representing intermediate steps in the data pipeline also won't be exported (same as e.g. `ImageTensor`), so a short name is fine I think.